Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs

```diff
@@ -0,0 +1,276 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Meadow.Hardware;
+using Meadow.Peripherals.Sensors;
+using Meadow.Units;
+
+namespace Meadow.Foundation.Sensors.Environmental
+{
+    /// <summary>
+    /// Base class for SCD4x series of C02 sensors
+    /// </summary>
+    public abstract partial class Scd4x : ByteCommsSensorBase<(Concentration? Concentration, 
+                                                        Units.Temperature? Temperature,
+                                                        RelativeHumidity? Humidity)>,
+        ITemperatureSensor, IHumiditySensor
```

Comment:
Dont we have a IConcentrationSensor?

Reply:
We don't and I was on the fence about adding it ... I'll do it quickly now